tools/ddsrouter_tool/test/application/CMakeLists.txt

```diff
@@ -138,6 +138,7 @@ if(PYTHONINTERP_FOUND)
                                     "--exe" $<TARGET_FILE:ddsrouter_tool>
                                     "--config-file" ${CMAKE_CURRENT_BINARY_DIR}/configurations/${TEST}.yaml
                                     "--debug"
+                                    "--version"
```

Comment:
These are python test flags, no ddsrouter arguments. I do not think this is required here